app/views/taxons/show.html.erb

```diff
@@ -62,7 +62,7 @@
               }
             %>
             <%= content_tag(:div, data: ga4_data) do %>
-              <%= render(partial: section[:partial_template], locals: { section: section }) %>
+              <%= render(partial: section[:partial_template], locals: { section: section, disable_ga4: true }) %>
```

Comment:
Putting a `false` default into the partials seems sensible, but is there a situation where that isn't overridden by this line?

Reply:
I've not seen any other uses of the section partials, so not essential right now, my initial option was to hardcode `disable_ga4: true` in each section partial, happy to go with this approach if preferred

Reply:
I think that might be simpler - otherwise it implies that this is necessary because some other part of the code doesn't pass `true`, which isn't correct. We can always extend it later if needed 👍

Reply:
Agreed, thanks Andy, I've made this change